pkg/configurator/methods.go

```diff
@@ -3,10 +3,16 @@ package configurator
 import (
 	"encoding/json"
 	"fmt"
+	"time"
 
 	"github.com/openservicemesh/osm/pkg/constants"
 )
 
+const (
+	// defaultServiceCertValidityDuration is the default validity duration for service certificates
+	defaultServiceCertValidityDuration = 24 * time.Hour
```

Comment:
semi-related: Is `constants.go` expected to be used only if multiple modules need to read values? I imagine the declaration here is closed inside configurator and we do not expect someone else to read

Reply:
That is correct.